lib/reporters/html/partials/aggregations.hbs

```diff
@@ -0,0 +1,56 @@
+{{#if folder}}
```

Comment:
@czardoz Partialization has been done in order to account for deeply nested folder structures. The same structure can effectively be recursively re-used to make folder divisions.

As for folder name placement, keeping them separated from the request titlebar makes it possible to collapse entire folders, something that would be cumbersome with the `request / folder` approach.